### PR TITLE
BaSP-TG-12: Admin List and Delete

### PR DESCRIPTION
**Pull request by Baldizan**

### CODE REVIEW

File: src/Components/Admins/modal.module.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    transform: scale(1.10);
+    transition: all .2s ease-in-out;
+}
+
+.confirm__button {
+    margin: 1rem 0 2rem 0.50rem;
+    padding: 0.50rem 0.80rem;
+    font-weight: 500;
+    color: #fff;
+    background-color: #F13312;;
```
Comment (martumoser) [inline src/Components/Admins/modal.module.css:43]: You have two semicolon at the end of the line. Please delete one

File: src/Components/Admins/modal.module.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+.bottom__close__button:hover,
+.bottom__close__button:focus {
+    cursor: pointer;
+    transform: scale(1.015);
+    transition: all .2s ease-in-out;
+}
+
+.content {
+    color: #373867;
+    max-width: 35ch;
```
Comment (martumoser) [inline src/Components/Admins/modal.module.css:74]: why you are using ch unit?

File: src/Components/Admins/modal.module.css [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    font-size: 16px;
+    color: #373867;
+}
+
+.top__close__button:hover,
+.top__close__button:focus {
+    cursor: pointer;
+    transform: scale(1.10);
+    transition: all .2s ease-in-out;
+}
```
Comment (martumoser) [inline src/Components/Admins/modal.module.css:36]: are this properties necessary? they are the same of lines 48 to 53. Please put all together

File: src/Components/Admins/modal.module.css [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+.header {
+    display: flex;
+    justify-content: space-between;
+    width: 90%;
+    border-bottom: 1px solid rgba(0, 0, 0, 0.153);
+  }
+
+  .heading {
```
Comment (martumoser) [inline src/Components/Admins/modal.module.css:19]: Fix indentation

File: src/Components/Admins/index.js [hunk 5]
```diff
@@ -1,9 +1,96 @@
 import styles from './admins.module.css';
+import React, { useState, useEffect } from 'react';
+import Modal from './modal';
 
 function Admins() {
```
Comment (martumoser) [inline src/Components/Admins/index.js:5]: Replace for const and arrow function

File: src/Components/Admins/index.js [hunk 6]
```diff
@@ -1,9 +1,96 @@
...
+    saveAdmins(updatedAdminList);
+  };
+
   return (
     <section className={styles.container}>
-      <h2>Admins</h2>
+      <h2>Admin list</h2>
+      <button>Add new admin +</button>
+      <table className={styles.table}>
+        <col span="5" className={styles.columns} />
```
Comment (martumoser) [inline src/Components/Admins/index.js:31]: Avoid use col element. Use css properties to add styles to the table

File: src/Components/Admins/index.js [hunk 7]
```diff
@@ -1,9 +1,96 @@
...
+    const updatedAdminList = admins.filter((admin) => admin._id !== adminId);
+    saveAdmins(updatedAdminList);
+  };
+
   return (
     <section className={styles.container}>
-      <h2>Admins</h2>
+      <h2>Admin list</h2>
+      <button>Add new admin +</button>
+      <table className={styles.table}>
```
Comment (martumoser) [inline src/Components/Admins/index.js:30]: Table and rows should be another react component as was showing on the class

**Review by martumoser — CHANGES_REQUESTED** [review]
Please add a description to the PR

File: src/Components/Admins/modal.module.css (hunk 2 above)
Comment (Baldizan) [reply src/Components/Admins/modal.module.css:74]: .content is a <p> element, and "ch" makes determines the amount of charachters per line. Additionally, although responsiveness was not one of the task's requirements, "ch" enhances it.

File: src/Components/Admins/modal.module.css (hunk 3 above)
Comment (Baldizan) [reply src/Components/Admins/modal.module.css:36]: The scale is different for one and the other, because it does not look good otherwise.

**Comment by martumoser** [comment]
pr descartada. Resuelto en en TG-11:Admin add and delete

**Pull request closed without merging**